native/src/main/java/io/ballerina/stdlib/http/api/HttpResource.java

```diff
@@ -227,6 +268,12 @@ public static HttpResource buildHttpResource(MethodType resource, HttpService ht
         BMap resourceConfigAnnotation = getResourceConfigAnnotation(resource);
 
         if (checkConfigAnnotationAvailability(resourceConfigAnnotation)) {
+            if (resourceConfigAnnotation.getStringValue(NAME) != null) {
```

Comment:
We could use `Objects.nonNull` to check whether a particular object is null or not.

Reply:
Fixed in [a8c835e](https://github.com/ballerina-platform/module-ballerina-http/pull/1068/commits/a8c835e645db41d31e5f041b2fc786856153dbb3)